tests/cluster/run.tcl

```diff
@@ -15,6 +15,7 @@ proc main {} {
     spawn_instance redis $::redis_base_port $::instances_count {
         "cluster-enabled yes"
         "appendonly yes"
+        "enable-debug-command yes"
```

Comment:
let's also enable the protected configs.
we did remove all protections from `asserts/default.conf` too.
we can keep modules disabled sine the current infra doesn't support modules in sentinel and modules anyway.